catalog/systemd.pl.catalog.in

```diff
@@ -542,3 +542,39 @@ Support: %SUPPORT_URL%
 
 Po raz pierwszy podczas obecnego uruchomienia uzyskano synchronizację NTP
 i zainicjowano regulację lokalnego zegara systemowego.
+
+-- 7db73c8af0d94eeb822ae04323fe6ab6
+Subject: Początkowe zwiększenie zegara
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Zwiększono zegar systemowy na podstawie pliku ze znacznikiem czasu na dysku
+w celu zapewnienia, że nadal jest w przybliżeniu monotoniczny — nawet między
+ponownymi uruchomieniami — jeśli zegar czasu rzeczywistego jest niedostępny
+lub zawodny.
+
+-- 3f7d5ef3e54f4302b4f0b143bb270cab
+Subject: Rozszerzono PCR układu TPM
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Rejestr konfiguracji platformy (PCR) @PCR@ układu TPM, w rzędach @BANKS@,
+został rozszerzony ciągiem „@MEASURING@”.
+
+Kiedy system przechodzi do nowego etapu uruchamiania, podany PCR jest
+rozszerzany innym ciągiem, aby zapewnić, że zasady zabezpieczeń dla haseł
+związanych z TPM i innych zasobów są ograniczone do konkretnych etapów
+uruchamiania.
+
+-- f9b0be465ad540d0850ad32172d57c21
+Subject: Obniżono pamięć
```

Comment:
"Obniżono pamięć" → "Obniżono zużycie pamięci" ?

Reply:
Sure, that works.